db/migrate/20150515163602_create_mc_licenses.rb

```diff
@@ -0,0 +1,23 @@
+class CreateMcLicenses < ActiveRecord::Migration
+
+  TABLE_NAME = :mc_licenses
+
+  # Drops `mc_licenses`
+  #
+  # @return [void]
+  def down
+    drop_table TABLE_NAME
+  end
+
+  # Creates `mc_licenses`
+  #
+  # @return [void]
+  def up
+    create_table TABLE_NAME do |t|
+      t.string :abbreviation, null: false, unique: true
```

Comment:
`unique: true` here is ignored and does not create an index.  (Look at the diff for `db/schema.rb`.). Do the following instead:

```ruby
  # Creates `mc_licenses`
  #
  # @return [void]
  def up
    create_table TABLE_NAME do |t|
      t.string :abbreviation, null: false
      t.text :summary, null: false
      t.string :url, null: false
    end

    change_table TABLE_NAME do |t|
      t.index :abbreviation, unique: true
      t.index :summary, unique: true
      t.index :url, unique: true
    end
  end
```